### PR TITLE
fix(l10n): Add context note for table footers

### LINKED ISSUE #37641: Explicit "Total rows summary" for translators
This is not explicit enough for translators to do their job easily and accurately.

https://github.com/nextcloud/server/blob/16975ae45720945776155f026835cfdaf8491383/apps/files/src/components/FilesListTableFooter.vue#L25
Could we please get a more explicit wording or a context note to help us?

Thank you in advance
J

PS: forgive me if using the feature request template leads to wrong tagging, but the bug report template is a real pain for such small reports with so many mandatory non-relevant fields

### PR DESCRIPTION
**Pull request by Pytal**
## Summary

Fix https://github.com/nextcloud/server/issues/37641

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/files/src/components/FilesListTableFooter.vue [hunk 1]
```diff
@@ -22,6 +22,7 @@
 <template>
 	<tr>
 		<th class="files-list__row-checkbox">
+			<!-- TRANSLATORS Label for a table footer which summarizes the columns of the table -->
```
Comment (ShGKme) [inline apps/files/src/components/FilesListTableFooter.vue:25]: I might be wrong, but I thought, SFC files doesn't support `TRANSLATORS` comments. Neither in template nor in script...

It only has the path to the file and line number.

Comment (skjnldsv) [reply apps/files/src/components/FilesListTableFooter.vue:25]: image
https://github.com/nextcloud/server/blob/2d6cd78c3299fe343637f08d780ecb2646b33172/apps/settings/src/components/Users/NewUserModal.vue#L151-L155

Seems something is broken, but it's partially implemented?

Comment (skjnldsv) [reply apps/files/src/components/FilesListTableFooter.vue:25]: https://github.com/nextcloud/docker-ci/blob/85e124865de782afc516d79ac6301ef8b0a2da3b/translations/translationtool/src/translationtool.php#L75

Comment (skjnldsv) [reply apps/files/src/components/FilesListTableFooter.vue:25]: On it

Comment (skjnldsv) [reply apps/files/src/components/FilesListTableFooter.vue:25]: https://github.com/nextcloud/docker-ci/pull/619

**Review by skjnldsv — APPROVED** [review]

**Review by Valdnet — APPROVED** [review]

**Review by ShGKme — APPROVED** [review]

**Pull request merged**